build.gradle

```diff
@@ -206,7 +206,8 @@ subprojects {
 
   dependencyCheck {
     failBuildOnCVSS = 0
-    skipConfigurations += ['checkstyle']
+    skipConfigurations += ['checkstyle', 'spotbugs']
```

Comment:
Sry for limited knowledge but why skip spotbugs

Reply:
I don't mind this change, but it enlarges a bit the scope of this PR.

Reply:
Upgrading the plugin somehow triggered checking the `spotbugs` configuration for vulnerabilities when it never has before.  These two configurations are here because we don't care about vulnerabilities in the dependencies in spotbugs/checkstyle or their dependencies since they don't end up in any of our stuff.